vault/resource_aws_auth_backend_cert.go

```diff
@@ -0,0 +1,189 @@
+package vault
+
+import (
+	"encoding/base64"
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/hashicorp/vault/api"
+)
+
+var (
+	awsAuthBackendCertBackendFromPathRegex = regexp.MustCompile("^auth/(.+)/config/certificate/.+$")
+	awsAuthBackendCertNameFromPathRegex    = regexp.MustCompile("^auth/.+/config/certificate/(.+$)")
+)
+
+func awsAuthBackendCertResource() *schema.Resource {
+	return &schema.Resource{
+		Create: awsAuthBackendCertCreate,
+		Read:   awsAuthBackendCertRead,
+		Delete: awsAuthBackendCertDelete,
+		Exists: awsAuthBackendCertExists,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"cert_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of the certificate to configure.",
+				ForceNew:    true,
+			},
+			"aws_public_cert": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Base64 encoded AWS Public key required to verify PKCS7 signature of the EC2 instance metadata.",
+				ForceNew:    true,
+				/*DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					return base64.StdEncoding.EncodeToString([]byte(old)) == new ||
+						base64.RawStdEncoding.EncodeToString([]byte(old)) == new ||
+						base64.URLEncoding.EncodeToString([]byte(old)) == new
+				},*/
```

Comment:
I'd rather not have commented-out code here since it is invariably confusing for future maintainers, but if there's a good reason to keep this then let's at least have a comment above it explaining what it does, why it's commented out, and in what situation we might un-comment it in future.

Reply:
My bad, just a failure to clean up before push.